packages/back-end/src/app.ts

```diff
@@ -133,763 +135,819 @@ if (SENTRY_DSN) {
   );
 }
 
-if (!process.env.NO_INIT && process.env.NODE_ENV !== "test") {
-  init();
-}
-
 app.set("port", process.env.PORT || 3100);
 app.set("trust proxy", EXPRESS_TRUST_PROXY_OPTS);
-
 // Pretty print on dev
 if (ENVIRONMENT !== "production") {
   app.set("json spaces", 2);
 }
 
-app.use(cookieParser());
+if (stringToBoolean(process.env.PYTHON_SERVER_MODE)) {
+  // increase max payload json size to 50mb as a single query can return up to 3000 rows
+  // and we pass the results of all queries at once into python
+  app.use(
+    bodyParser.json({
+      limit: process.env.PYTHON_SERVER_INPUT_SIZE_LIMIT || "50mb",
+    })
+  );
+  app.use(compression());
+  app.use(httpLogger);
+  app.post("/stats", async (req, res) => {
+    try {
+      const results = await runStatsEngine(req.body);
```

Comment:
issue: I think we need to validate the payload here to ensure it matches what we expect

Reply:
This will not be accessible from the internet.  Also stats_server.py does validating.